Tasks/Maven/mavencodeanalysis.ts

```diff
@@ -163,14 +158,16 @@ function createAndUploadBuildSummary(enabledTools:string[], modules:ma.ModuleAna
     });
 
     // Add a double space and a final line with descriptive text, if there were any violations to be reported
-    if (totalViolationsInBuild > 0) {
+    if (getTotalViolationsInModules(modules) > 0) {
         buildSummaryLines.push("");
         buildSummaryLines.push("Code analysis results can be found in the 'Artifacts' tab.");
     }
 
     // Save and upload build summary
     // Looks like: "PMD found 13 violations in 4 files.  \r\n
-    // FindBugs found 10 violations in 8 files."
+    // FindBugs found 10 violations in 8 files.  \r\n
+    //   \r\n
```

Comment:
Line returns. There's a specific combination of two spaces and then a line return (a markdown newline) which I have to specify this way - a multiline string tends to cut one of the two out.